lib/peeps.rb

```diff
@@ -0,0 +1,49 @@
+require 'pg'
+
+class Peeps
+  attr_reader :peep
+
+def self.all
+if ENV['ENVIROMENT'] == 'test'
+  connection = PG.connect(dbname: 'chitter_test')
+  #this determines which database to connect to
+else
+  connection = PG.connect(dbname: 'chitter')
+end
+
+ result = connection.exec("SELECT * FROM peeps")
+ #once you are in chitter return all peeps stored in table
+ #this is exactly how you input when using the database
+ result.map{ |peep| peep['message'] }
+ #u need to call the value of the hash! hence [message]
+end
+def self.create(message:)
+  if ENV['ENVIROMENT'] == 'test'
+    connecttion = PG.connect(dbname: 'chitter_test')
+  else
+    connection = PG.connect(dbname: 'chitter')
+  end
+result = connection.exec("INSERT INTO peeps (message) VALUES ('#{message}')")
+  #connection.exec("INSERT INTO peeps (message) values ('This is a peep!');")
+ end
+
+def self.date
+  if ENV['ENVIROMENT'] == 'test'
+    connection = PG.connect(dbname: 'chitter_test')
+    #this determines which database to connect to
+  else
+    connection = PG.connect(dbname: 'chitter')
+  end
+   result = connection.exec("SELECT * FROM peeps ORDER BY DESC")
```

Comment:
It's great that you're thinking about the date here. 

How would the database know what time the tweet was posted? Does the Peep table need to change at all?